dotCMS/src/main/java/com/dotcms/util/VoidDelegate.java

```diff
@@ -0,0 +1,38 @@
+package com.dotcms.util;
+
+
+import com.dotmarketing.exception.DotDataException;
+
+/**
+ * <p>
+ *     A delegate class is basically a class that executes a specific task, it is usually used to decouple some generic task from the specific one (the delegate will be the specialization in the routine).
+ * </p>
+ * <p>
+ * So for instance you may have a Delegate to perform a Quartz Job receiving an specific information from the Quartz. A  Delegate is a class that can be registered to a Quartz Job. Delegate
+ * classes will receive basic information from such a Job that will allow them to perform any custom action.
+ * For example, a delegate can connect to a running service and notify a
+ * component that one or more new events have occurred. This way, other services
+ * and even UI components can react to the new information and provide a useful
+ * output to the user.
+ * </p>
+ *
+ * <p>
+ * You can also used a Delegate to perform a task in a thread or anything other generic algorithm routine that needs some specific and unknown part.
+ * </p>
+ *
+ * This delegate does not return anything nor requires anything.
+ * 
+ * @author Jose Castro
+ * @version 3.7
+ * @since Jul 13, 2016
+ *
+ */
+public interface VoidDelegate<T> {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') T is not used in the interface. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2326)